api2-war/src/main/java/eu/europeana/api2/v2/web/controller/SearchController.java

```diff
@@ -390,53 +418,60 @@ public ModelAndView searchJsonGet(
+        if (addBrackets) {
             filterQuery.append(")");
         }
         return filterQuery.toString();
     }
 
     /**
      * Processes all qf parameters. Note that besides returning a new array of refinements we may add new filterTags to
-     * the provided filterTags list (if there are image, sound, video or mimetype refinements)
+     * the provided filterTags list (if there are image, audio, video or mimetype refinements)
      */
-    private String[] processQfParameters(String[] refinementArray, Boolean media, Boolean thumbnail, Boolean fullText,
-                                         Boolean landingPage, List<Integer> filterTags) {
-        boolean      hasImageRefinements            = false;
-        boolean      hasSoundRefinements            = false;
-        boolean      hasVideoRefinements            = false;
-        FacetEncoder facetEncoder = new FacetEncoder();
-        List<String> newRefinements                 = new ArrayList<>();
-
-        Set<MimeTypeEncoding>   imageMimeTypeRefinements = new HashSet<>();
-        Set<MimeTypeEncoding>   soundMimeTypeRefinements = new HashSet<>();
-        Set<MimeTypeEncoding>   videoMimeTypeRefinements = new HashSet<>();
-        Set<MimeTypeEncoding>   otherMimeTypeRefinements = new HashSet<>();
-        Set<ImageSize>          imageSizeRefinements = new HashSet<>();
-        Set<ImageAspectRatio>   imageAspectRatioRefinements = new HashSet<>();
-        Set<AudioDuration>      soundDurationRefinements = new HashSet<>();
-        Set<VideoDuration>      videoDurationRefinements = new HashSet<>();
-        Set<ImageColorSpace>    imageColourSpaceRefinements = new HashSet<>();
-        Set<VideoQuality>       videoHDRefinements = new HashSet<>();
-        Set<AudioQuality>       soundHQRefinements = new HashSet<>();
+    private String[] processQfParameters(String[] refinementArray,
+                                         Boolean media,
+                                         Boolean thumbnail,
+                                         Boolean fullText,
+                                         Boolean landingPage,
+                                         List<Integer> filterTags) {
+        boolean      hasImageRefinements = false;
+        boolean      hasAudioRefinements = false;
+        boolean      hasVideoRefinements = false;
+        boolean      hasTextRefinements  = false;
+        boolean      hasOtherRefinements = false;
+        FacetEncoder facetEncoder        = new FacetEncoder();
+        List<String> newRefinements      = new ArrayList<>();
+
+        Set<MimeTypeEncoding>   imageMimeTypeRefinements      = new HashSet<>();
+        Set<MimeTypeEncoding>   audioMimeTypeRefinements      = new HashSet<>();
+        Set<MimeTypeEncoding>   videoMimeTypeRefinements      = new HashSet<>();
+        Set<MimeTypeEncoding>   textMimeTypeRefinements       = new HashSet<>();
+        Set<MimeTypeEncoding>   otherMimeTypeRefinements      = new HashSet<>();
```

Comment:
Please delete unused variable

Reply:
Indeed! That one was an artefact of the previous unclarity re. textual tech facets. It's no longer needed and I removed it.